### PR TITLE
Add v1.1.0 of mattermost-plugin-incident-management to the Marketplace

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Add Incident Management to the marketplace.

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2422,6 +2422,58 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
```
Comment (lieut-data) [inline plugins.json:2426]: Not an issue given redirects, but we should update our `plugin.json` in v1.2 given the rename.

**Review by lieut-data — APPROVED** [review]

**Comment by lieut-data** [comment]
Adding @mickmister alongside @hanzei -- hoping to get a sign off from integrations for this change today.

Adding @prapti to work alongside @DHaussermann so we can qualify this today as well.

File: plugins.json [hunk 2]
```diff
@@ -2422,6 +2422,58 @@
...
+      "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": []
+      }
+    },
+    "enterprise": false,
```
Comment (lieut-data) [inline plugins.json:2460]: I think we need to use the `--enterprise` flag so that this gets set to true and only gets shown for E20 customers.

Comment (cpoile) [reply plugins.json]: done!

File: plugins.json (hunk 1 above)
Comment (cpoile) [reply plugins.json:2426]: agreed

**Review by lieut-data — APPROVED** [review]

**Review by mickmister — APPROVED** [review]
LGTM :tada:

**Review by prapti — APPROVED** [review]
Tested on local server. LGTM!

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@cpoile @lieut-data Is this ready to getting merged?

**Comment by cpoile** [comment]
yep

**Pull request merged**